data/timesketch.conf

```diff
@@ -26,13 +26,13 @@ SQLALCHEMY_DATABASE_URI = 'postgresql://<USERNAME>:<PASSWORD>@localhost/timesket
 # Make sure that the Elasticsearch server is properly secured and not accessible
```

Comment:
I imagine we should also change those lines WDYT?

Reply:
Done